install/kubernetes/helm/istio/charts/grafana/templates/create-custom-resources-job.yaml

```diff
@@ -65,7 +65,7 @@ spec:
       serviceAccountName: istio-grafana-post-install-account
       containers:
         - name: hyperkube
-          image: "{{ .Values.global.hyperkube.repository }}:{{ .Values.global.hyperkube.tag }}"
+          image: "{{ .Values.global.hyperkube.hub }}/hyperkube:{{ .Values.global.hyperkube.tag }}"
```

Comment:
Some comments for the hardcode image name here, sometimes, people may also want to re-name the docker image as well, and for such case, they have to change the helm chart, how about make the image name configurable as well? 

Just want to have some discussion here and I'm OK to merge this first.

Reply:
@costinm @sdake comments for this?

Reply:
@gyliu513 I'm not opposed to adding an image KV pair for each of the changes in this PR, but would prefer to do as a followon as this has gone through review and testing already.

Reply:
sure, thanks @sdake